### ISSUE TITLE
search lookahead is more accurate and does not match results on second query

### ISSUE THREAD

**Issue by ashleygwilliams** [npm/newww#1892]
video: http://d.pr/v/1dCRh

## steps to reproduce
1. type `gulp` in search bar
2. see look ahead
<img width="409" alt="screen shot 2016-02-19 at 1 39 06 pm" src="https://cloud.githubusercontent.com/assets/1163554/13185303/5b29dc2c-d70e-11e5-9447-2d6c4aacd2e1.png">
3. hit enter, see results
<img width="582" alt="screen shot 2016-02-19 at 1 39 15 pm" src="https://cloud.githubusercontent.com/assets/1163554/13185292/5067f936-d70e-11e5-9e31-e7b51fa43801.png">

4. hit enter again, see different results
<img width="521" alt="screen shot 2016-02-19 at 1 39 36 pm" src="https://cloud.githubusercontent.com/assets/1163554/13185285/4ad544e2-d70e-11e5-846f-494ebe118d1b.png">

**Comment by leo** [comment]
Same here!

**Comment by aredridel** [comment]
Correct: the search drop-down is third party and uses a different algorithm.

**Comment by joepie91** [comment]
I personally find the results in the search drop-down to be far, *far* more accurate than the results after submitting the search form, almost all of the time.
